packages/common/package.json

```diff
@@ -51,13 +51,15 @@
     "require-context": "^1.1.0",
     "solidity-coverage": "^0.7.13",
     "truffle-deploy-registry": "^0.5.1",
-    "web3": "^1.3.5"
+    "web3": "^1.3.5",
+    "ethers": "^5.1.3"
   },
   "devDependencies": {
     "@eth-optimism/hardhat-ovm": "^0.2.2",
     "@nomiclabs/hardhat-etherscan": "^2.1.4",
     "@nomiclabs/hardhat-truffle5": "^2.0.0",
     "@nomiclabs/hardhat-web3": "^2.0.0",
+    "@nomiclabs/hardhat-ethers": "^2.0.2",
```

Comment:
Since we import `hardhat-deploy`, we should install `@nomiclabs/hardhat-ethers` the [hardhat-deploy recommended way](https://github.com/wighawag/hardhat-deploy#npm-install-hardhat-deploy). This arises from the fact that `hardhat-deploy` forks `hardhat-ethers` to add some extra features to the `deploy` task.

Reply:
done. I removed the previous dependency then ran:

```
yarn workspace @uma/common add @nomiclabs/hardhat-ethers@npm:hardhat-deploy-ethers ethers --dev
```

and updated the import.